src/understand-tidb/execution.md

```diff
@@ -1 +1,27 @@
 # Execution
+
+The `executor` package contains most of the codes related to execution. The input of the executor is a plan tree of the query returned from the planner, and the output of the executor is the result of the query.
```

Comment:
I'd prefer you mention the entrypoint of the input and the output data structure, so that contributors can navigate among the codebase.